misago/users/admin/tests/test_users.py

```diff
@@ -10,6 +10,7 @@
 from ...utils import hash_email
 
 User = get_user_model()
+PASSWORD = 'Lor3mIpsum'
```

Comment:
Please make sure that tests are using testing config. Changing password shouldn't be necessary.